src/insanely_fast_whisper/cli.py

```diff
@@ -91,11 +91,13 @@
 def main():
     args = parser.parse_args()
 
+    dtype = torch.float32 if args.device_id == "cpu" else torch.float16
```

Comment:
Added this because otherwise fails on i5 CPU:
```
RuntimeError: Input type (torch.FloatTensor) and weight type (torch.HalfTensor) should be the same or input should be a MKLDNN tensor and weight is a dense tensor
```